### PR TITLE
[CPM-66] Seed course dummy data

### PR DESCRIPTION
**Pull request by daltonlim**

### CODE REVIEW

File: seed-db/scripts/index.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
+
 const yargs = require("yargs");
 const fs = require("fs");
 const { generateCourses } = require("./course");
+const { generateRegulations } = require("./regulation");
 
 const filename = "./fake-courses.json";
+const regulations_filename = "./fake-regulations.json";
```
Comment (FocalChord) [inline seed-db/scripts/index.js:8]: can we use camel case please

File: seed-db/scripts/index.js [hunk 2]
```diff
@@ -15,10 +18,13 @@ const argv = yargs
     .help()
     .alias("help", "h").argv;
 
+function writeFile (filename,obj) {
+    const jsonObj = JSON.stringify(obj);
+    fs.writeFileSync(filename, jsonObj);
+}
+
```
Comment (FocalChord) [inline seed-db/scripts/index.js:25]:
```suggestion
const writeFile = (filename,obj) => {
    const jsonObj = JSON.stringify(obj);
    fs.writeFileSync(filename, jsonObj);
}

```

Could we use es6 functions to keep up with the convention used throughout the application please

File: seed-db/scripts/index.js (hunk 1 above)
Comment (daltonlim) [reply seed-db/scripts/index.js]: Updated

File: seed-db/scripts/course.js [hunk 3]
```diff
@@ -1,17 +1,51 @@
...
 };
 
-const createCourse = () => {
+getRandomElement = (array) =>{
+    return array[Math.floor(Math.random() * array.length)]
+}
+
+const points = [15]
+const semesters = ["S1", "S2", "SS"]
+const subjects = ["SOFTENG","COPMSCI","ENGGEN"]
```
Comment (wryao64) [inline seed-db/scripts/course.js:17]: Typo in COMPSCI

File: seed-db/scripts/regulation.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    return array[Math.floor(Math.random() * array.length)]
+}
+const points = [45,60,100]
+const pointRequirement = ["UPTO", "EXACT", "ATLEAST"]
+
+const createRegulation = () => {
+    const regulation = {
+        points: getRandomElement(points),
+        pointRequirement: getRandomElement(pointRequirement),
+    }
```
Comment (wryao64) [inline seed-db/scripts/regulation.js:12]: Please add some dummy courses

**Review by wryao64 — CHANGES_REQUESTED** [review]

**Review by Richardng12 — APPROVED** [review]
LGTM

File: seed-db/scripts/course.js [hunk 5]
```diff
@@ -1,17 +1,52 @@
...
+        courseList.push(getRandomElement(courses))
+    }
+    return courseList
+}
+const regulationsArray = regulations.generateRegulations(10)
+const createCourse = (i) => {
     // Make this smarter
     const course = {
-        courseName: faker.name.findName(),
+        _id: "5f4e14016282ae3706ba54f" + i,
```
Comment (FocalChord) [inline seed-db/scripts/course.js:40]: What is this id for lol?

Comment (daltonlim) [reply seed-db/scripts/course.js:40]: regulations which require course _id in the schema

Comment (FocalChord) [reply seed-db/scripts/course.js:40]: For the actual course object, this would be randomised right?

Comment (daltonlim) [reply seed-db/scripts/course.js:40]: yea

**Review by FocalChord — APPROVED** [review]
lgtm

File: seed-db/scripts/course.js (hunk 3 above)
Comment (daltonlim) [reply seed-db/scripts/course.js]: updated

**Pull request merged**